@here/harp-datasource-protocol/lib/InterpolatedProperty.ts

```diff
@@ -101,7 +101,9 @@ export function getPropertyValue(property: Value | Expr | undefined, env: Env):
         return property.evaluate(env, ExprScope.Dynamic);
     }
 
-    if (typeof property !== "string") {
+    if (typeof property === "function") {
```

Comment:
Please update typings as well.

With this patch `Expr` cannot possibly exist in technique property in renderer (only! *), so first `if` is unnecessary ... but agin only in renderer.

Also it would be good to have same representation of dynamic attrs in renderer and decoder - possibly compile them same way in SSE - at least for "public" usage outside of SSE.

Reply:
what do you mean with "Also it would be good to have same representation of dynamic attrs in renderer and decoder - possibly compile them same way in SSE - at least for "public" usage outside of SSE." ?

Reply:
I mean that in SSE compiles technique attribute to `Value | Expr` and decoder can observe these types.
In rendering phase technique attr after `compileTechniques` is `Value | function`.

Again 
 - 1) It breaks typings, we don't expect `function` there
 - 2) It breaks `Expr.isExpr` in many places to check if we really need to update these parameters

Reply:
all the Expr.isExpr should definitely go, we should change the type to `Value | Expr | function` (as you already mentioned) and maybe introduce a isDynamic(v) that should return true if v is an Expr or a function.